hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/upgrade/TestHDDSUpgrade.java

```diff
@@ -286,6 +286,7 @@ private void testPostUpgradeConditionsDataNodes() {
               return false;
             }
           } catch (IOException e) {
+            LOG.error("Exception. ", e);
```

Comment:
loosely related but we have a typo in this file later on, in line 994, Failuer is written in the method name instead of Failure, should we fix it here?

Reply:
Fixed.